ISICbalance/ViewControllers/AccountViewController.swift

```diff
@@ -1,41 +1,68 @@
 //
-//  AccountView.swift
+//  AccountViewController.swift
 //  ISICbalance
 //
 //  Created by Rostislav Babáček on 11/03/2019.
 //  Copyright © 2019 Rostislav Babáček. All rights reserved.
 //
 
 import UIKit
+import SnapKit
+import SwiftKeychainWrapper
 import ACKategories
 
-class AccountView: UIView {
+class AccountViewController: BaseViewController {
+    private var viewModel: AccountViewModel
```

Comment:
viewModely v 99 % případů chceš mít `private let`. Nechceš v průběhu života controlleru měnit viewModel, ve spojení s reaktivním observingem by to byla sebevražda